src/main/resources/db/changelog/changeset/create-albums-table.xml

```diff
@@ -0,0 +1,30 @@
+<databaseChangeLog
```

Comment:
Make rebase on already existing changes to get liquibase from already merged commit